setup.cfg

```diff
@@ -38,6 +36,7 @@ install_requires =
     nglview
     MDAnalysis
     utm
+    dask
```

Comment:
is it a new dependency?

Reply:
it was specified in the requirements.txt (by @fabiocruz, [here](https://github.com/inductiva/inductiva/commit/1f958923cb81d9cc54310ce3e57d43e97fe80fc2)), so i added it to the `setup.cfg` because probably it was added there by mistake

Reply:
according to @fabiocruz it is needed